uvm/impl/com/untangle/uvm/NetworkManagerImpl.java

```diff
@@ -1788,6 +1789,13 @@ private void sanitizeNetworkSettings( NetworkSettings networkSettings )
                 intf.setV4PPPoERootDev(intf.getSystemDev());
                 intf.setSystemDev("ppp" + pppCount);
                 intf.setSymbolicDev("ppp" + pppCount);
+                //Encrypt the password for v4PPPoEPassword and save it in v4PPPoEPasswordEncrypted
+                if(StringUtils.isNotBlank(intf.getV4PPPoEPassword())){
+                    intf.setV4PPPoEPasswordEncrypted(PasswordUtil.getEncryptPassword(intf.getV4PPPoEPassword()));
+                }
+                if(StringUtils.isNotBlank(intf.getV4PPPoEPasswordEncrypted())){
```

Comment:
Do you need to check again notblank on `v4PPPoEPasswordEncrypted` ?

Reply:
If password is already encrypted, then also we need to keep the v4PPPoEPassword as empty.
Other wise the v4PPPoEPassword field get removed from network.js file